web/lang/ru_ru.php

```diff
@@ -51,7 +50,7 @@
 // do this by default, uncomment this if required.
 //
 // Example
-header( "Content-Type: text/html; charset=utf-8" );
+header( "Content-Type: text/html; charset=koi8-r" );
```

Comment:
Denis if you want to update your 1.26 file to current github master version and then make some changes still you can still submit changes. I can see some of the changes are good.
Your english is better than my russian, so thank you for contributing.